### ISSUE TITLE
unmet dependencies installing php5-xcache from php 5.6 for wheezy

### ISSUE THREAD

**Issue by useltmann** [gplessis/dotdeb-php#125]
when i want to install php5-xcache from php56 package repository for debian wheezy i get unmet dependencies:
```
$ apt-get install php5-xcache
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 php5-xcache : Depends: phpapi-20100525
E: Unable to correct problems, you have held broken packages.

**Comment by rraptorr** [comment]
There is no xcache available for PHP 5.6 in Dotdeb repository. PHP 5.6 comes with opcache builtin, which is an alternative for xcache which performs better and is supported by the PHP team.

**Comment by useltmann** [comment]
although it might be true that opcache should be favored to xcache as opcode cache xcache also provides a variable cache api, on which some projects might rely on. and since opcache is in php since version 5.5 and there are no problems installing xcache next to opcache in php 5.5 from dotdeb repository i do not see the problem doing so in php5.6, or am i wrong?

**Comment by rraptorr** [comment]
No, there is no problem, do as you wish.
As stated above xcache is not available in Dotdeb for php 5.6 anyway (the error you get is because you are trying to install php5-xcache from wheezy, which is for php 5.4).

**Comment by rraptorr** [comment]
Also, if you need variable cache you can use apcu, which is available from Dotdeb for php 5.6.

**Comment by gplessis** [comment]
@useltmann : as @rraptorr mentioned kindly, xcache packages are not available starting PHP 5.6, sorry.
